conduit/pipeline/pipeline.go

```diff
@@ -59,16 +61,28 @@ type pipelineImpl struct {
 	pipelineMetadata state
 }
 
+var (
```

Comment:
When I change `var` to `const` I get an error: `errors.New("pipeline stopped") (value of type error) is not constant` and that makes sense because constants need to be defined at compile-time. On the other hand, wrapping them inside of functions won't work either, since I want to be able to use `errors.Is()` with them.